features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+      | required | fields        | Parameter bevat een porpertynaam met onjuiste casing. |
+
+    Scenario: opvragen veld met fields door opgeven naam veld die meerdere keren in de resource voorkomt
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende geboorte gegevens
+      | naam                                      | waarde   |
+      | datum (01.03.10)                          | 19860401 |
+      | land.code                                 | 6030     |
+      | plaats.code                               | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                   |
+      | type                | RaadpleegMetBurgerservicenummer          |
+      | burgerservicenummer | 999994086                                |
+      | fields              | code                                     |
+      Dan bevat de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En bevat de response een invalidParams[0] met de volgende gegevens
+      | code     | name          | reason              |
+      | required | fields        | Deel van de parameterwaarde niet correct: geboorte.land.code  |
+      En bevat de response een invalidParams[1] met de volgende gegevens
+      | code     | name          | reason              |
+      | required | fields        | Deel van de parameterwaarde niet correct: geboorte.plaats.code  |
+
+  Rule: De response bevat alleen de properties die in de fields parameter gevraagd en indien relevant de volgende properties :
+        De volgende velden worden geleverd ongeacht of de fields parameter is gebruikt en ongeacht de eventueel gevraagde velden in de fields parameter:
+        - inOnderzoek wordt meegegeven wanneer gevraagde corresponderende velden in onderzoek zijn. Zie in_onderzoek.feature voor uitleg wanneer attributen in onderzoek zijn.
```

Comment:
Ik stel voor dat we de wisselwerking tussen inOnderzoek en fields opnemen in de in_onderzoek.feature. Daar kan je hier dan naar verwijzen. Dus niet alleen verwijzen voor uitleg wanneer een attribuut in onderzoek is, maar ook verwijzen voor leveren van inOnderzoek in combinatie met fields.

Reply:
besproken. wordt uitgebreid beschreven in inOnderzoek feature. @JohanBoer verwijderd uit fields feature en neemt daarin verwijzing op naar inOnderzoek feature